ovs_container/ovs_container_start.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+/usr/share/openvswitch/scripts/ovs-ctl start
+ovs-vsctl set-manager ptcp:5678
```

Comment:
you should pass it as args (default ot 5678)

Reply:
Done